final_task/rss_reader/action_functions.py

```diff
@@ -0,0 +1,179 @@
+""" Module of creation functions and action functions.
+
+    Functions:
+    create_logger(com_line_args) -> logger
+    get_com_line_args() -> com_line_args
+    get_news(command_line_args, logger) -> news_collection
+    print_news_stdout(news_collection) -> None
+    print_news_json(news_collection) -> None
+    print_news(news_collection, com_line_args, logger) -> None
+    print_cache_news(news_collection, logger) -> None
+    print_cache_news_json(news_collection, logger) -> None
+    convert_date(date_str, logger) -> str_date
+    clean_str(string) -> clean_string """
```

Comment:
Here and below: there is actually no need to write function list in module docstring.